designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/ScenarioStatistics.scala

```diff
@@ -174,14 +210,25 @@ object ScenarioStatistics {
     else orderedList.last
   }
 
+  def mapNameToStat(name: String): String = {
```

Comment:
IMHO we should not rely on hashCode. We should better simply remove vowels or calculate and shorten md5 or sha hash.